src/main/java/gregtech/common/blocks/foam/BlockPetrifiedFoam.java

```diff
@@ -20,7 +20,7 @@ public BlockPetrifiedFoam(boolean isReinforced) {
         setSoundType(SoundType.SNOW);
         setResistance(isReinforced ? 4.0f : 16.0f);
         setHardness(isReinforced ? 1.0f : 4.0f);
-        setCreativeTab(GregTechAPI.TAB_GREGTECH);
```

Comment:
Neither foam, petrified foam, or reinforced stone actually appears in the creative tab, so perhaps this could be fixed.

Reply:
Probably because there were no item representation at all, so I think it's working correctly. I'll remove the creative tab decl for both foams and solid foams to prevent any confusion.